content/en/docs/setup/production-environment/tools/kubeadm/install-kubeadm.md

```diff
@@ -51,6 +51,33 @@ may [fail](https://github.com/kubernetes/kubeadm/issues/31).
 If you have more than one network adapter, and your Kubernetes components are not reachable on the default
 route, we recommend you add IP route(s) so Kubernetes cluster addresses go via the appropriate adapter.
 
+## Ensure iptables tooling does not use the nftables backend
+
+In Linux, nftables is available as a modern replacement for the kernel's iptables subsystem. The
+`iptables` tooling can act as a compatibility layer, behaving like iptables but actually configuring
+nftables. This nftables backend is not compatible with current Kubernetes releases: it causes duplicated
+firewall rules and breaks `kube-proxy`.
+
+If your system's `iptables` tooling uses the nftables backend, you will need to switch the `iptables`
+tooling to 'legacy' mode to avoid these problems. This is the case on at least Debian 10 (Buster),
+Ubuntu 19.04, RHEL and CentOS 8, Fedora 29 and newer releases of these distributions by default.
+
+{{< tabs name="iptables_legacy" >}}
+{{% tab name="Ubuntu, Debian or HypriotOS" %}}
+```bash
+update-alternatives --set iptables /usr/sbin/iptables-legacy
+update-alternatives --set ip6tables /usr/sbin/ip6tables-legacy
+update-alternatives --set arptables /usr/sbin/arptables-legacy
+update-alternatives --set ebtables /usr/sbin/ebtables-legacy
+```
+{{% /tab %}}
+{{% tab name="CentOS, RHEL or Fedora" %}}
+```bash
+update-alternatives --set iptables /usr/sbin/iptables-legacy
```

Comment:
Is this possible on RHEL8?

Reply:
@joeabbey it is definitely possible in Fedora: `
# ls -l /usr/sbin/iptables
lrwxrwxrwx. 1 root root   26 Aug  2 13:47 /usr/sbin/iptables -> /etc/alternatives/iptables
`# update-alternatives --list | grep iptables
iptables                auto    /usr/sbin/iptables-legacy
` so I would imaging RHEL8 as well.

Reply:
Fedora 29 and 30 have iptables 1.8. Earlier versions have 1.6.x and only support legacy.